docs/src/rules/no-warning-comments.md

```diff
@@ -101,6 +105,56 @@ Examples of **correct** code for the `{ "terms": ["todo", "fixme", "any other te
 
 :::
 
+### Decoration Characters
+
+Examples of **incorrect** code for the `{ "decoration": "*" }` options:
+
+::: incorrect
+
+```js
+/*eslint no-warning-comments: ["error", { "decoration": "*" }]*/
+
+//***** todo decorative asterisks are ignored *****//
+/**
+ * TODO new lines and asterisks are also ignored in block comments.
+ */
+```
+
+:::
+
+Examples of **incorrect** code for the `{ "decoration": ["/", "*"] }` options:
+
+::: incorrect
+
+```js
+/*eslint no-warning-comments: ["error", { "decoration": ["/", "*"] }]*/
+
+////// TODO decorative slashes and whitespace are ignored //////
+//***** todo decorative asterisks are also ignored *****//
+/**
+ * TODO new lines are also ignored in block comments.
+ */
+```
+
+:::
+
+Examples of **correct** code for the `{ "decoration": ["/", "*"] }` options:
+
+::: correct
+
+```js
+/*eslint no-warning-comments: ["error", { "decoration": ["/", "*"] }]*/
+
+// This is to do
+/**
+ * The same goes for block comments
+ * with any other interesting term
+ * or fix me this
+ */
```

Comment:
I think that examples where terms appear after some non-decorative characters would be more representative as correct examples for this option.